xapian-letor/ranker/svmranker.cc

```diff
@@ -41,8 +41,8 @@ namespace Xapian {
 
 class XAPIAN_VISIBILITY_DEFAULT SVMRanker: public Ranker {
 
-    string model = null;
-    double[] weight;
+    string model = NULL;
+    double* weight;
```

Comment:
These changes should be in a separate commit, because they aren't related to moving the code around.